defaults/main.yml

```diff
@@ -0,0 +1,23 @@
+xen_vman_vms:                   []
+xen_vman_memory:                1024
```

Comment:
It's not clear what this does. Please call it something like `xen_vman_default_memory`

Reply:
I think this goes for all of these variables. It is not clear to me whether a value here is a global xen setting or if it is just a default value if the vm configuration doesn't contain this specific value.